### PR TITLE
Sonar Issues

### PR DESCRIPTION
**Pull request by bekeles**

### CODE REVIEW

**Comment by bekeles** [comment]
closes #92 

Fabian I fixed the major issues reported by sonar. I am not sure if we need a change log for this or not.

File: src/main/java/com/imsweb/mph/MphConstants.java [hunk 1]
```diff
@@ -243,23 +242,26 @@ public class MphConstants {
     }
 
     public static final Map<String, String> BENIGN_BRAIN_2004_HISTOLOGY_GROUPING;
+    private static final String EPENDYMOMAS = "Ependymomas";
```
Comment (depryf) [inline src/main/java/com/imsweb/mph/MphConstants.java:245]: Private static should start with an underscore.

**Review by depryf — CHANGES_REQUESTED** [review]

**Comment by depryf** [comment]
Just a minor thing. No need of a changelog entry for this type of code cleanup.

**Comment by bekeles** [comment]
@depryf I know you are busy, I just don't want this to slip away from your way :)

**Pull request merged**